team/team.go

```diff
@@ -32,6 +40,38 @@ func LoadTeams(fluidkeysDirectory string) ([]Team, error) {
 	return teams, nil
 }
 
+// Save validates the team then tries to create a toml team roster in a subdirectory of the given
+// directory. If successful it returns the filepath of the written toml file
+func (t *Team) Save(fluidkeysDirectory string) (string, error) {
```

Comment:
Function is great, but it I don't think the `Team` should know about fluidkeys directory. I propose it should be something like `team.Save(newTeam)`